_layouts/documentation.html

```diff
@@ -5,33 +5,38 @@
 <main class="usa-layout-docs usa-section" id="main-content">
   <div class="grid-container">
     {% if page.sidenav %}
-      <div class="grid-row grid-gap">
-        <aside class="usa-layout-docs__sidenav desktop:grid-col-3 position-sticky tablet-lg:height-viewport">
-          <nav>
-            <ul class="usa-sidenav">
-              {% include nav/list.html
-                links = page.sidenav
-                li_class = 'usa-sidenav__item'
-                ul_class = 'usa-sidenav__sublist'
-              %}
-            </ul>
-          </nav>
-        </aside>
-
-        <div class="usa-layout-docs__main desktop:grid-col-9 usa-prose">
+      <div class="grid-row grid-gap padding-x-4">
```

Comment:
I notice the homepage content is not quite aligned correctly with the navbar.

![image](https://github.com/18F/identity-dev-docs/assets/1779930/17b2ff3e-303a-4f09-8111-8b9f757bb233)

It looked fine on the documentation pages, which I tracked back to this change.

I think we should ideally be able to avoid the explicit padding here and use the native container widths instead. 

Doing some testing, I had a little better luck using the markup from [the design system Extended Header example](https://designsystem.digital.gov/components/header/#extended-header), with a `usa-navbar` but _not_ a `usa-nav-container` as container element.

Reply:
@aduth  removing the grid-container div seemed to do the trick for being able to get rid of all the padding adjustments (just had to figure out which max width setting I was missing in the sass). let me know if that looks ok now!